conversion.go

```diff
@@ -87,3 +88,7 @@ func (z *Int) SetFromBig(b *big.Int) bool {
 	}
 	return overflow
 }
+
```

Comment:
perhaps copy the docstring from big.Int ?

Reply:
Done.